tests/clients/test_experiment.py

```diff
@@ -329,6 +341,277 @@ def test_experiment_run_data_schema_multiple():
     assert data == {"tags": [TAG_BODY, OTHER_TAG_BODY]}
 
 
+PROJECT_ID_FILTER = SingleFilter(
+    SingleFilterBy.PROJECT_ID, None, SingleFilterOperator.EQUAL_TO, PROJECT_ID
+)
+PROJECT_ID_FILTER_BODY = {
+    "by": "projectId",
+    "key": None,
+    "operator": "eq",
+    "value": str(PROJECT_ID),
+}
+
+EXPERIMENT_ID_FILTER = SingleFilter(
+    SingleFilterBy.EXPERIMENT_ID, None, SingleFilterOperator.EQUAL_TO, 1
+)
+EXPERIMENT_ID_FILTER_BODY = {
+    "by": "experimentId",
+    "key": None,
+    "operator": "eq",
+    "value": "1",
+}
+
+RUN_ID_FILTER = SingleFilter(
+    SingleFilterBy.RUN_ID,
+    None,
+    SingleFilterOperator.EQUAL_TO,
+    EXPERIMENT_RUN_ID,
+)
+RUN_ID_FILTER_BODY = {
+    "by": "runId",
+    "key": None,
+    "operator": "eq",
+    "value": str(EXPERIMENT_RUN_ID),
+}
+
+DELETED_AT_FILTER = SingleFilter(
+    SingleFilterBy.DELETED_AT, None, SingleFilterOperator.EQUAL_TO, DELETED_AT
+)
+DELETED_AT_FILTER_BODY = {
+    "by": "deletedAt",
+    "key": None,
+    "operator": "eq",
+    "value": DELETED_AT_STRING_PYTHON,
+}
+
+TAG_FILTER = SingleFilter(
+    SingleFilterBy.TAG, "tag_key", SingleFilterOperator.EQUAL_TO, "tag_value"
+)
+TAG_FILTER_BODY = {
+    "by": "tag",
+    "key": "tag_key",
+    "operator": "eq",
+    "value": "tag_value",
+}
+
+PARAM_FILTER = SingleFilter(
+    SingleFilterBy.PARAM,
+    "param_key",
+    SingleFilterOperator.EQUAL_TO,
+    "param_value",
+)
+PARAM_FILTER_BODY = {
+    "by": "param",
+    "key": "param_key",
+    "operator": "eq",
+    "value": "param_value",
+}
+
+METRIC_FILTER = SingleFilter(
+    SingleFilterBy.METRIC,
+    "metric_key",
+    SingleFilterOperator.EQUAL_TO,
+    "metric_value",
+)
+METRIC_FILTER_BODY = {
+    "by": "metric",
+    "key": "metric_key",
+    "operator": "eq",
+    "value": "metric_value",
+}
+
+AND_FILTER = CompoundFilter(
+    operator=CompoundFilterOperator.AND,
+    conditions=[
```

Comment:
To shorten this (and `AND_FILTER_BODY`, `OR_FILTER`, etc.) you could just use the already existing single filters like `TAG_FILTER` to compose compound filters.